openapi.yaml

```
@@ -0,0 +1,50 @@
+openapi: 3.0.0
+info:
+  title: Zoning API
+  description: |-
+    An API for serving data related to the zoning of New York City.
+  version: 0.1.0
+servers:
+  - url: https://zoning.planningdigital.com/api
+tags:
+  - name: Boroughs
+paths:
+  /boroughs:
+    get: 
+      summary: List boroughs
+      operationId: getBoroughs
+      tags:
+      - Boroughs
+      responses:
+        '200':
+          description: An array of borough objects.
+          content:
+            application/json:
+              schema: 
+                type: array
+                items: 
+                  $ref: '#/components/schemas/Borough'
+components:
+  schemas:
+    Borough:
+      type: object
+      properties:
+        id:
+          type: string
+          description: A single character numeric string containing the common number used to refer to the borough. Possible values are 1-5.
+          pattern: \b[1-9]\b
+          example: 2
```

This does not technically matter but will drive me crazy if I don't mention it: Manhattan is 1 (Bronx is 2).
```suggestion
          example: 1
```